topic_admin.go

```diff
@@ -0,0 +1,74 @@
+package kazoo
+
+import (
+	"errors"
+	"fmt"
+)
+
+var (
+	ErrTopicExists          = errors.New("Topic already exists")
+	ErrTopicMarkedForDelete = errors.New("Topic is already marked for deletion")
+)
+
+// CreateTopic creates a new kafka topic with the specified parameters and properties
```

Comment:
Is this synchronous? Can you immediately start producing/consuming after creating a topic? If not, should we provide a way to have a way to wait until it becomes available?